themes/bootstrap3/templates/myresearch/menu.phtml

```diff
@@ -2,7 +2,7 @@
 <?php foreach ($this->menu as $menuGroup): ?>
   <h3 id="<?=$this->idPrefix ?? ''?><?=$menuGroup['id']?>"><?=$this->transEsc($menuGroup['label'])?></h3>
   <nav class="myresearch-menu" aria-labelledby="<?=$this->idPrefix ?? ''?><?=$menuGroup['id']?>">
-    <ul <?=$menuGroup['class'] ? 'class="' . $menuGroup['class'] . '"' : ''?>>
+    <ul <?=isset($menuGroup['class']) ? 'class="' . $menuGroup['class'] . '"' : ''?>>
```

Comment:
I'd prefer `!empty()` so that setting the class to an empty string doesn't cause trouble. And perhaps, while at it, put the space inside the condition so that we can avoid a trailing space when there's no class. :)